v1/proto/service/gribi.proto

```diff
@@ -421,3 +427,49 @@ message GetResponse {
   // A set of entries that are installed on the device.
   repeated AFTEntry entry = 1;
 }
+
+// FlushRequest is the request sent from the client to the server when a Flush
+// RPC is called.
+message FlushRequest {
+  // election specifies the election ID from a client. It is valid only
```

Comment:
if specified in ALL_PRIMARY should `election` be ignored by the server ? or should it reject the request